drivers/metal/inflection_map.h

```diff
@@ -0,0 +1,125 @@
+/**************************************************************************/
+/*  inflection_map.h                                                      */
```

Comment:
Since it's header only template, it might be worth moving to `core/templates`, to be reusable by other code.